recipes/ncl/yum_requirements.txt

```diff
@@ -0,0 +1,8 @@
+csh
+imake
+libx11-devel
+libXaw-devel
+libXmu-devel
+byacc
+flex
+flex-devel
```

Comment:
I tried using `flex` from `conda-forge`, but it did not provide libl.so. I tried changing our code to build against libfl.so instead, but that didn't work either, which is why I opted to use yum to install flex.